aea/manager.py

```diff
@@ -337,6 +376,7 @@ def add_agent(
             agent_name=agent_name,
             agent_overrides=agent_overrides,
             component_overrides=component_overrides,
+            config=config,
```

Comment:
Not ok, checks need to be added so config or overrides can be passed, not both

Reply:
Done.